### DISCUSSION TITLE
水を差す example sentence is keyed on the wrong meaning

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#55] [Errors and Corrections]
This entry as well is keyed on the first meaning but it should be on the second meaning.

First meaning: "to pour water (into) / to dilute"

Second meaning: (idiom) "to hinder / to hamper / to put a damper on (etc)" 

---

JP: 人の話に水を差さないでくれ。

EN: Don't throw a wet blanket over our conversation.

---

also if I want to be bit nitpicky, I don't like the expression "throw a wet blanket over" and I'd prefer "put a damper on" as it matches the already existing entry more, but that's just my personal preference.

**Comment by stephenmk** [comment]
Thanks, I updated the index to key the sentence to the second sense. The translation is a little colorful but I don't expect it will cause any confusion. The five glosses that precede the example sentence should be enough to get the point across.
